repos/fdbt-site/src/pages/api/service.ts

```diff
@@ -17,8 +17,27 @@ export default async (req: NextApiRequestWithSession, res: NextApiResponse): Pro
         }
 
         const dataSource = getRequiredSessionAttribute(req, TXC_SOURCE_ATTRIBUTE).source;
+        const fareTypeAttribute = getRequiredSessionAttribute(req, FARE_TYPE_ATTRIBUTE);
+        const isReturn = 'fareType' in fareTypeAttribute && ['period', 'return'].includes(fareTypeAttribute.fareType);
 
         const service = await getServiceByIdAndDataSource(getAndValidateNoc(req, res), serviceId, dataSource);
+        const directions = Array.from(
+            service.journeyPatterns.reduce((set, pattern) => {
+                set.add(pattern.direction);
+                return set;
+            }, new Set<string>()),
+        );
+
+        if (directions.length === 1 && isReturn) {
```

Comment:
What will happen here if we have outbound + clockwise (or a pairing which both mean the same thing)? That would mean there are multiple directions in this array, but they both mean the same thing.

Reply:
Yeah good point, we need to check that they have inbound + outbound below 👍

Reply:
this is done